crates/weaver_forge/src/jq.rs

```diff
@@ -75,15 +78,19 @@ pub fn execute_jq(
     let ctx = Ctx::new(values, &inputs);
 
     // Bundle Results
-    let mut errs = Vec::new();
-    let mut values = Vec::new();
-    let filter_result = filter.run((ctx, Val::from(input.clone())));
-    for r in filter_result {
-        match r {
-            Ok(v) => values.push(serde_json::Value::from(v)),
-            Err(e) => errs.push(e),
-        }
-    }
+    let mut values = filter
+        .run((ctx, Val::from(input.clone())))
+        .collect::<Result<Vec<_>, _>>()
+        .map_err(|e| Error::FilterError {
```

Comment:
So this returns first error right now (via `?`) vs. all of them.  I think this is a fine improvement, but we may want to consider collecting all errors, returning them ALL immediately across all filters, but this is fine as an initial fix.

Reply:
I see, that's why we had this logic to collect errors! let me see if I can fix it